apps/app_manager.py

```diff
@@ -4,8 +4,7 @@
 import traceback
```

Comment:
This whole rewrite will have to be removed - it only undoes the changes that I recently did, because of some Git shenanigans =)